buildchain/buildchain/image.py

```diff
@@ -304,11 +304,11 @@ def show() -> str:
         build_args={'NGINX_IMAGE_VERSION': NGINX_IMAGE_VERSION, 'NODE_IMAGE_VERSION': NODE_IMAGE_VERSION},
         task_dep=['_image_mkdir_root'],
         file_dep=(
```

Comment:
This commit should be squashed with the previous one.

Reply:
done